### PR TITLE
Sentry integration

### PR DESCRIPTION
**Pull request by ohvitorino**
This PR removes Errbit and replaces it with Sentry ([https://sentry.io](https://sentry.io)).

### CODE REVIEW

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -33,8 +33,8 @@ parameters:
     # Form configuration
     secret: ThisTokenIsNotSoSecretChangeIt
 
-    # Errbit configuration
-    errbit_api_key:
+    #Sentry configuration
+    sentry.dsn: ''
```
Comment (tijsverkoyen) [inline app/config/parameters.yml.dist]: Can you add a task in AC to alter the `sumo project:*`-commands.

File: src/SumoCoders/FrameworkCoreBundle/Resources/doc/deployment/deployment.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ Depending on the environment you should give other answers:
 * locales: enter an array with the languages that should be available
 * locale: enter the default locale
 * secret: this should be a random string
-* errbit_api_key: the api key you have created for this project
+* sentry.dsn: the sentry api key you have created for this project
```
Comment (tijsverkoyen) [inline src/SumoCoders/FrameworkCoreBundle/Resources/doc/deployment/deployment.md]: Is this an API key? Maybe add a link to sentry?

File: app/config/parameters.yml.dist (hunk 1 above)
Comment (ohvitorino) [inline app/config/parameters.yml.dist]: Done.

File: src/SumoCoders/FrameworkCoreBundle/Resources/doc/deployment/deployment.md (hunk 2 above)
Comment (ohvitorino) [inline src/SumoCoders/FrameworkCoreBundle/Resources/doc/deployment/deployment.md]: Will do.

Comment (ohvitorino) [inline src/SumoCoders/FrameworkCoreBundle/Resources/doc/deployment/deployment.md]: e8b373a

**Pull request merged**